.azure-pipelines/azure-pipelines.yml

```diff
@@ -177,6 +177,8 @@ stages:
               test: rhel/8.3@3.6
             - name: RHEL 8.3 py38
               test: rhel/8.3@3.8
+            - name: RHEL 8.4 py38
+              test: rhel/8.4@3.8
```

Comment:
Replace the existing 8.3 tests with 8.4 in our test matrix instead of adding new entries.

Reply:
There's another set of 8.3 entries to update under `Remote` (these are `Incidental Remote`).